go.mod

```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
-	github.com/cosmos/cosmos-sdk v0.44.5
+	github.com/cosmos/cosmos-sdk v0.45.1
```

Comment:
Is this a breaking change or not?

Reply:
It is a breaking change. Which is not an issue in my opinion because changes in codebase are breaking changes too and require an upgrade proposal anyway.

> Cosmos SDK v0.45.0 is a logical continuation of the v0.44.* series, but brings a couple of state- and API-breaking changes requested by the community.